### ISSUE TITLE
lookup error exception in /jinja2/environment.py" for apt_key module since last weeks commits

### ISSUE THREAD

**Issue by darKoram** [ansible/ansible#4709]
Yesterday I synched to the latest ansible head.
ad5731b4a80355dbf5aa26db495f60e2e88131e1

When I run the ansible-hadoop playbook 
I now get failing errors below.  Sorry I don't have any 
experience with apt_key to produce a minimal example here,
but the error msg looks fairly useful.

```

---
\#in file: roles/cdh_common/tasks/main.yml

- name: Make sure Cloudera CDH4 repository is included
  apt_repository: repo='deb [arch=amd64] http://archive.cloudera.com/cdh4/ubuntu/precise/amd64/cdh precise-cdh4 contrib'
  tags:
    - hadoop
    - hbase

- name: Make sure Cloudera CDH4 key is present
  apt_key: url=http://archive.cloudera.com/cdh4/ubuntu/precise/amd64/cdh/archive.key
  tags:
    - hadoop
    - hbase
```

```
TASK: [cdh_common | Make sure Cloudera CDH4 key is present] ******************* 
fatal: [Ubuntu-Cluster-04] => Traceback (most recent call last):
  File "/Users/kbroughton/vcp/git/mentat/ansible/lib/ansible/runner/__init__.py", line 377, in _executor
    exec_rc = self._executor_internal(host, new_stdin)
  File "/Users/kbroughton/vcp/git/mentat/ansible/lib/ansible/runner/__init__.py", line 464, in _executor_internal
    return self._executor_internal_inner(host, self.module_name, self.module_args, inject, port, complex_args=complex_args)
  File "/Users/kbroughton/vcp/git/mentat/ansible/lib/ansible/runner/__init__.py", line 655, in _executor_internal_inner
    result = handler.run(conn, tmp, module_name, module_args, inject, complex_args)
  File "/Users/kbroughton/vcp/git/mentat/ansible/lib/ansible/runner/action_plugins/normal.py", line 54, in run
    return self.runner._execute_module(conn, tmp, module_name, module_args, inject=inject, complex_args=complex_args)
  File "/Users/kbroughton/vcp/git/mentat/ansible/lib/ansible/runner/__init__.py", line 289, in _execute_module
    (remote_module_path, module_style, shebang) = self._copy_module(conn, tmp, module_name, args, inject, complex_args)
  File "/Users/kbroughton/vcp/git/mentat/ansible/lib/ansible/runner/__init__.py", line 838, in _copy_module
    in_path, complex_args, module_args, inject
  File "/Users/kbroughton/vcp/git/mentat/ansible/lib/ansible/module_common.py", line 141, in modify_module
    module_data = self._template_imports(module_data)
  File "/Users/kbroughton/vcp/git/mentat/ansible/lib/ansible/module_common.py", line 125, in _template_imports
    return t.render(vars_input)
  File "/Library/Python/2.7/site-packages/Jinja2-2.7-py2.7.egg/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/Library/Python/2.7/site-packages/Jinja2-2.7-py2.7.egg/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "<template>", line 83, in top-level template code
UndefinedError: 'lookup' is undefined
```

Previously I had been on commit:
d7e172f776a87fe79a6cda47ffe93409d38eecef
When i revert to it, things are fine.
I noticed that there were some mods to the apt module since then.

**Comment by jimi-c** [comment]
Could you please test with the latest head of devel from today? There were other reports of the lookup plugin having issues but they seem to be resolved.

**Comment by jctanner** [comment]
Bug confirmed in ad5731b4a80355dbf5aa26db495f60e2e88131e1 
Fix confirmed in 0e9184eefd4eb1729814aad3ee00fdef2fbc2ee4
